pages/common/git-restore.md

```diff
@@ -1,17 +1,33 @@
 # git restore
 
 > Restore working tree files. Requires git version 2.23+.
-> See also `git checkout`.
+> See also `git checkout` and `git reset`.
 > More information: <https://git-scm.com/docs/git-restore/>.
 
-- Restore a deleted file from the contents of the current commit (HEAD):
+- Restore an unstaged file to the version of the current commit (HEAD):
 
 `git restore {{path/to/file}}`
 
-- Restore a file to a version from a different commit:
+- Restore an unstaged file to the version of a specific commit:
 
 `git restore --source {{commit}} {{path/to/file}}`
 
-- Undo any uncommitted changes to tracked files, reverting to the current HEAD:
+- Discard all unstaged changes to tracked files:
 
-`git restore .`
+`git restore :/`
+
+- Unstage a file:
+
+`git restore --staged {{path/to/file}}`
+
+- Unstage all files:
+
+`git restore --staged :/`
+
+- Discard all changes to files, staged or not:
+
+`git restore --worktree --staged :/`
+
+- Interactively select parts of files to restore:
```

Comment:
Does this allow the user to choose which files to restore, or which sections of files to restore?

Reply:
which sections of files. git calls those 'hunks', but i think 'parts' or 'sections' is easier to understand.

Reply:
it's similar to git add --patch

Reply:
I agree.  How about changing it to 'sections' to make a bit clearer?
```suggestion
- Interactively select sections of files to restore:
```